pkg/upgrader/upgrade/kubernetes_binaries.go

```diff
@@ -59,10 +59,18 @@ curl -L "https://github.com/containernetworking/plugins/releases/download/{{ .CN
 
 RELEASE="v{{ .KUBERNETES_VERSION }}"
 
-sudo mkdir -p /opt/bin
-cd /opt/bin
+
+
+sudo mkdir -p /var/tmp/kube-binaries
```

Comment:
Can we maybe use `/tmp`? Does it exist on CoreOS out of box?

Reply:
I tested it on our machines, and yes it does.